src/app/rundown-view/components/rundown-tooltips/video-hover-scrub/video-hover-scrub.component.ts

```diff
@@ -0,0 +1,107 @@
+import { Component, ElementRef, Input, OnDestroy, OnInit, ViewChild } from '@angular/core'
+import { Tv2PieceType } from 'src/app/core/enums/tv2-piece-type'
+import { PieceLifespan } from 'src/app/core/models/piece-lifespan'
+import { Observable, Subject, takeUntil } from 'rxjs'
+import { ConfigurationService } from 'src/app/shared/services/configuration.service'
+import { StudioConfiguration } from 'src/app/shared/models/studio-configuration'
+import { IconButton, IconButtonSize } from 'src/app/shared/enums/icon-button'
+
+const ROUNDING_PRECISION: number = 100
+
+@Component({
+  selector: 'sofie-video-hover-scrub',
+  templateUrl: './video-hover-scrub.component.html',
+  styleUrls: ['./video-hover-scrub.component.scss'],
+})
+export class VideoHoverScrubComponent implements OnInit, OnDestroy {
+  @Input() public fileName: string
+  @Input() public type: Tv2PieceType
+  @Input() public tooltipHoverElementWidth: number
+  @Input() public tooltipHoverMouseEventObservable: Observable<MouseEvent | undefined>
+  @Input() public pieceLifespan?: PieceLifespan
+  @Input() public playedDurationInMs?: number
+  @Input() public isMediaUnavailable?: boolean
+
+  public currentVideoTimeInS: number
+  public videoDurationInS: number
```

Comment:
Why are they in seconds? Everywhere else we use milliseconds.
If you want to show the value to the user in seconds, then just use a Pipe to transform them from milliseconds to seconds when displaying them.

Reply:
Done